boards/enjoydigital/litex_vexriscv/litex_vexriscv.dts

```diff
@@ -235,6 +235,79 @@
 			};
 		};
 
+		sdhc0: sdhc@e000e800 {
+			compatible = "litex,litesdcard-sdhc";
+			interrupt-parent = <&intc0>;
+			reg = <0xe000e800 0x4>,
```

Comment:
Wondering if we can use one base address here. Or are all these register offsets configurable in the RTL?

Reply:
no see other answer